data/templates/dnsmasq/plain/resolv.dnsmasq.conf

```diff
@@ -0,0 +1,26 @@
+# According to dnsmasq's doc, dnsmasq will automatically 
+# choose the fastest-to-answer resolver as primary 
+# (if strict-order is disabled)
+
+# List taken from
+# http://diyisp.org/dokuwiki/doku.php?id=technical:dnsresolver
```

Comment:
we should probably inform these organisations that their DNS server is listed here